packages/devtools_app/lib/src/charts/flutter/flame_chart.dart

```diff
@@ -724,6 +739,11 @@ class FlameChartNode<T> {
     @required bool hovered,
     @required double zoom,
   }) {
+    final zoomedWidth = rect.width * zoom;
+    if (zoomedWidth < 0.5) {
```

Comment:
Add a TODO: we should probably still show a solid line although perhaps fading it out.
0.5 seems a bit arbitrary for this case. It could probably equally reasonably be 0.2 for example.

Reply:
added TODO